treeherder/intermittents_commenter/commenter.py

```diff
@@ -169,6 +169,10 @@ def print_or_submit_changes(self, all_bug_changes):
                 # sleep between comment submissions to avoid overwhelming servers
                 time.sleep(1)
 
+        if settings.COMMENTER_API_KEY is None:
+            # confirm number of comments when on stage/dev
```

Comment:
With the conditional removed, this comment is repeating what the code does, so we can probably skip it